src/app/project/contact-form/contact-form.component.html

```diff
@@ -1,51 +1,52 @@
-            <div *ngIf="project.contactFormFilesEnabled">
-              <label class="control-label" for="files">Attach File(s)</label>
-              <app-file-upload [maxFiles]="3" [maxSize]="10" [files]="contactFormFiles" (filesChange)="manageFileChange($event)"></app-file-upload>
-            </div>
+        <div class="form-group">
+          <label class="control-label first" for="name">Name*</label>
+          <input class="form-control" type="text" name="name" id="name" required ngModel />
+          <label class="control-label" for="email">Email Address*</label>
+          <input class="form-control" type="text" name="email" id="email" required ngModel />
+          <label class="control-label" for="message">Message*</label>
+          <textarea class="form-control" name="message" id="message" required ngModel></textarea>
+          <div *ngIf="project.contactFormFilesEnabled">
+            <label class="control-label" for="files">Attach File(s)</label>
+            <app-file-upload [maxFiles]="3" [maxSize]="10" [files]="contactFormFiles"
+              (filesChange)="manageFileChange($event)"></app-file-upload>
           </div>
+        </div>
       </section>
       <section class="m-b-0">
-          <h3>Notice of Collection</h3>
-          <div *ngIf="!project.collectionNotice">
-            <p>This information is collected by Ministry of Forests, Lands Natural Resource Operations and Rural Development under
-              Section 26(c) of the <a class="paragraph-link" href="http://www.bclaws.ca/Recon/document/ID/freeside/96165_00" target="_blank">
+        <h3>Notice of Collection</h3>
+        <div *ngIf="!project.collectionNotice">
+          <p>This information is collected by Ministry of Water, Lands and Resource Stewardship under
+            Section 26(c) of the <a class="paragraph-link"
+              href="http://www.bclaws.ca/Recon/document/ID/freeside/96165_00" target="_blank">
               Freedom of Information and Protection of Privacy Act</a> and will be used to provide you with email
-              updates. Should you have any questions about the collection of your personal information, please contact:
-            </p>
-            <p class="m-b-0">Provincial Stewardship Strategies and Planning Branch
-              <br>Ministry of Water, Land and Resource Stewardship
-              <br>PO Box 9525, STN PROV GOVT
-              <br>Victoria BC, V8W 9C3
-              <br><a class="paragraph-link" href="mailto:landuseplanning@gov.bc.ca">landuseplanning@gov.bc.ca</a>
-            </p>
-          </div>
-          <div *ngIf="project.collectionNotice" [innerHTML]="project.collectionNotice"></div>
+            updates. Should you have any questions about the collection of your personal information, please contact:
+          </p>
+          <p class="m-b-0">Provincial Stewardship Strategies and Planning Branch
+            <br>Ministry of Water, Land and Resource Stewardship
```

Comment:
I checked the address since we're updating this page. It looks like this address might be incorrect. When I look up the contact addresses for that ministry, I get different results.
https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/water-land-and-resource-stewardship/ministry-contacts

I looked up that P.O box and found the following contact:
https://dir.gov.bc.ca/gtds.cgi?show=Branch&organizationCode=WLRS&organizationalUnitCode=CSCEM

You might want to check with Steve quickly. We could update the address if needed.